.storybook/main.js

```diff
@@ -76,6 +76,15 @@ module.exports = {
       'emotion-theming': toPath('node_modules/@emotion/react'),
     };
 
+    config.resolve.fallback = {
+      fs: false,
+      crypto: false,
+      browser: false,
+      tls: false,
+      https: false,
+    };
+
     return config;
   },
+  staticDirs: ['./static', '../data'],
```

Comment:
This is how we're including the storybook static directory AND the simorgh data folder as a static directory (which is then available when the storybook app is running)